src/project.js

```diff
@@ -44,18 +52,37 @@ function project(location) {
       throw new Error(`${value} for ${field} is not valid.`);
     }
 
-    options[field] = Object.keys(target);
+    options[field] = Object.keys(forestType);
 
-    if (value && target[value]) {
-      target = target[value];
+    if (value && forestType[value]) {
+      forestType = forestType[value];
     } else {
       // Location does not provide any more values for conditions.
       break;
     }
   }
-  const newLocation = { ...location, options };
-  if (typeof target === 'string') {
-    newLocation.target = target;
+
+  if (typeof forestType !== 'string') {
+    throw new Error(
+      'No height Level found. Try with another Target height Level.',
```

Comment:
Is this a good error message? I guess this error is more general, more like "No projection found.". Please also add a test for this error.

Reply:
Yeah, good question. Modified it, now it should make more sense. Also added the test for this case. :smile: